### PR TITLE
Perform operations on pypi_project only when pypi is True.

### PR DESCRIPTION
**Pull request by Aniket-Pradhan**
Solves #215 .

### CODE REVIEW

File: release_bot/configuration.py [hunk 1]
```diff
@@ -140,12 +140,13 @@ def load_release_conf(self, conf):
 
         # HEADS UP: pypi_project is set as self's attribute, not returned in parsed_conf
         # Try to get name from release-conf.yaml first, if it fails try to parse setup.cfg
-        self.pypi_project = parsed_conf.get('pypi_project') or get_pypi_project_from_setup_cfg()
-        if self.pypi_project is None:
-            msg = "pypi_project is not set, falling back to repository_name"
-            self.logger.warning(msg)
-            # Set pypi_project to repository name by default
-            self.pypi_project = self.repository_name
+        if parsed_conf.get('pypi'):
```
Comment (jpopelka) [inline release_bot/configuration.py]: At this point `'pypi'` has to be in `parsed_conf`, so 
```suggestion
        if parsed_conf['pypi']:
```

Comment (Aniket-Pradhan) [reply release_bot/configuration.py]: Not necessarily. When `pypi == False` then it will not be in `parsed_conf`. Hence, it would give a key error.

Comment (jpopelka) [reply release_bot/configuration.py]: [This](https://github.com/user-cont/release-bot/blob/master/release_bot/configuration.py#L127) makes sure it's there.

Comment (Aniket-Pradhan) [reply release_bot/configuration.py]: When `pypi` is False, then according to [this](https://github.com/user-cont/release-bot/blob/master/release_bot/configuration.py#L129) `pypi` should not be in `parsed_conf`

Comment (jpopelka) [reply release_bot/configuration.py]: Ok, now I see, thanks. I'm just wondering why the [filtering](https://github.com/user-cont/release-bot/blob/master/release_bot/configuration.py#L129) is actually there :-)

Comment (Aniket-Pradhan) [reply release_bot/configuration.py]: Probably so that we don't pass the unneeded information around, and save some bytes? :P

**Pull request merged**